.lintstagedrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "*.ts": ["prettier --write", "eslint --fix"],
+  "*.js": ["prettier --write", "eslint --fix"],
```

Comment:
I added `--fix` based off the CLI docs here. `format:fix` also uses the `--fix` flag.